layers/+vim/evil-better-jumper/README.org

```diff
@@ -19,7 +19,17 @@ locations.
 * Install
 To use this configuration layer, add it to your =~/.spacemacs=. You will need to
 add =evil-better-jumper= to the existing =dotspacemacs-configuration-layers=
-list in this file.
+list in this file. If =C-i= inserts an indent instead of
+=better-jumper-jump-forward=, please use =M-x describe-variable= to check the value
+of the symbol =dotspacemacs-distinguish-gui-tab= and ensure it is set to true in
```

Comment:
remove "the symbol"

set to "t" not "true"

Reply:
Good catch